### PR TITLE
Lr4 dkv

### PR DESCRIPTION
**Pull request by DavletshinKirill**

### CODE REVIEW

File: README.md
Comment (DmtryErk) [inline README.md:1]: This is good, my friend. I accept it.

Comment (DavletshinKirill) [reply README.md:1]: thanks

File: ДКВ.txt
Comment (DmtryErk) [inline ДКВ.txt:1]: This document is good too. Well done!

**Review by DmtryErk — APPROVED** [review]

File: ДКВ.txt
Comment (DmtryErk) [inline ДКВ.txt:1]: This document is good.

**Pull request merged**